app/controllers/api/v1/merchants/random_controller.rb

```diff
@@ -0,0 +1,12 @@
+module Api
+  module V1
+    module Merchants
+      class RandomController < ApplicationController
+        def show
+          render json: Merchant.order("RANDOM()").first
```

Comment:
I'd like to figure out the ActiveRecord method for this at some point, if it exists.